cmd/site-replication.go

```diff
@@ -2217,6 +2217,9 @@ func getAdminClient(endpoint, accessKey, secretKey string) (*madmin.AdminClient,
 	if err != nil {
 		return nil, err
 	}
+	if globalBucketTargetSys.isOffline(epURL.Host, epURL.Scheme == "https") {
```

Comment:
Maybe it is little bit easier to have globalBucketTargetSys.isOffline(u url.URL) and use it everywhere

Reply:
updated - we just need the endpoint